src/components/ServiceSearch.jsx

```diff
@@ -8,11 +9,54 @@ import {
   OutlinedInput,
 } from "@material-ui/core";
 import SearchIcon from "@material-ui/icons/Search";
+import { makeStyles } from "@material-ui/core/styles";
+
 import Search from "../modules/Search";
-import { Redirect, useRouteMatch } from "react-router-dom";
-import { useSelector } from "react-redux";
+
+const useStyles = makeStyles((theme) => ({
+  searchBar: {
+    [theme.breakpoints.up("xs")]: {
+      height: "45.6px",
```

Comment:
accuracy, i like it. Does it need to be this accurate though?

Reply:
maybe but probably yes. otherwise the lines from the button don't align with the lines from the input field :)

Reply:
yes we need better solution for this. Not now but we definitely should fix this later